config/app_config.yml.SAMPLE

```diff
@@ -167,6 +167,13 @@ default: &defaults
   # default to allow automatically adding new articles on sync only when less than 200 articles in total
   #shared_supplier_article_sync_limit: 200
 
+# Multi_coop example configuration
+# In order to make this work set "multi_coop_install: true"
+#anotherfoodcoop:
+#  <<: *defaults
+#  database:
+#    database: foodsoft_anotherfoodcoop
+
```

Comment:
what's the benefit of adding this lines? IMO everything is already present in the file.